controllers/ctrl.c

```diff
@@ -117,6 +117,9 @@ void ctrl_set_mode(ctrl_t* ctrl, ctrl_mode_t new_mode)
     if (new_mode < CTRL_MODE_NUMOF) {
         ctrl->control.current_mode = new_mode;
         DEBUG("ctrl: New mode: %d\n", ctrl->control.current_mode);
+
+        if (ctrl->control.current_mode == CTRL_MODE_STOP)
```

Comment:
Maybe this would make sense to always set current_cycle to 0.
Anyway this feature is used for 1) multi-line log aggregation, 2) controller speed order pattern generation which are function of times (not part of this patchset)... need to give a try to check if no regression without this test.

Reply:
I agree